pkg/dashboard/static/actions.js

```diff
@@ -337,4 +337,21 @@ $("#btnRollback").click(function () {
 $("#btnAddRepository").click(function () {
     setHashParam("section", "repository")
     window.location.reload()
-})
\ No newline at end of file
+})
+
+$("#inputSearch").keyup(function() {
+    var val = $(this).val().toLowerCase();
+
+    $(".charts li").hide()
+    
+    $(".charts li").each(function(){
+        var chartNameElem = this.firstElementChild
+        var chartName = $(chartNameElem).text().toLowerCase()
+        if(chartName.indexOf(val) != -1) {
+            $(this).show()
+        }
+    })
+})
+window.onload = function() {
```

Comment:
What this is for? The control should be empty when the page loads.

Reply:
It wasn't clearing by itself on page refresh that is why I added this
Any other way to handle it?

Reply:
You should place this call into this block: https://github.com/komodorio/helm-dashboard/blob/main/pkg/dashboard/static/repo.js#L23

This way, the filter will clear each time the repository is switched. Or you can place it even under `loadRepoView()` to clear it only when navigation to "Repository" section happens. Although I don't see any logic in keeping filter between different repositories.

Reply:
Fixed, please help review

Reply:
Looks good